src/mrt.rs

```diff
@@ -229,6 +229,13 @@ impl Ticks {
     pub unsafe fn from_u32(value: u32) -> Self {
         Self(value)
     }
+
+    /// Returns the number of ticks of this `Tick` instance.
+    /// This method is provided as a fallback to avoid performance overhead.
+    /// You may also use the `Into` implementations instead, if you can afford it.
```

Comment:
Is that true? I don't think there should be any performance difference between this and the `Into` implementation (unless the compiler does something weird). I think the advantage of having the inherent method is for visibility (an `Into` implementation can be overlooked easily).

Reply:
To be honest I'm not quite sure what my train of thought was here. happy to open a PR that removes this comment